recipe/meta.yaml

```diff
@@ -0,0 +1,100 @@
+{% set name = "recommenders" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/microsoft/recommenders/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8db3785543b2ace6ddc68994e71c8920efa781432708d8d4cc7aff720f11b55f
+
+build:
+  number: 0
+  skip: True  # [py<38 or py>=310 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - wheel
```

Comment:
Seems like it wants numpy during setup:
https://github.com/microsoft/recommenders/blob/29c64c9e8e19073da70f89f8308f6e8a2bd9fca4/setup.py#L140

ANd 

https://github.com/microsoft/recommenders/blob/29c64c9e8e19073da70f89f8308f6e8a2bd9fca4/pyproject.toml#L5

Reply:
Added!